Modules/IO/MINC/src/itkMINCImageIO.cxx

```diff
@@ -459,6 +459,13 @@ MINCImageIO::ReadImageInformation()
   dir_cos.Fill(0.0);
   dir_cos.SetIdentity();
 
+  // Conversion matrix for RAS to LPS
```

Comment:
Please use unambiguous notations of "RightToLeft,AnteriorToPosterior..."  in the comment.